test/mockFunctions.spec.js

```diff
@@ -16,6 +17,18 @@ ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 
 describe('verifica as funções e os mocks', () => {
   // Crie suas mock functions aqui
+  mockFunctions.add.mockImplementation((x,y) => x + y);
+  mockFunctions.subtract.mockImplementation((x,y) => x - y);
+  mockFunctions.multiply.mockImplementation((x,y) => x * y);
+  mockFunctions.divide.mockImplementation((x,y) => x / y);
+  mockFunctions.power.mockImplementation((x,y) => Math.pow(x,y));
+  mockFunctions.factorial.mockImplementation( x => {
+    if ( x === 0 || x === 1) return 1;
+    for(let j = x -1; j >= 1; j--){
+      x *= j;
+    }
+    return x;
+  });
```

Comment:
Muito bom, só a título de curiosidade dá pra resumir essas linhas em uma única.
```suggestion
  mockFunctions.factorial.mockImplementation((a) => (a === 0) ? 1 : a * mockFunctions.factorial(a - 1));
```